Source/WebCore/crypto/CryptoAlgorithm.h

```diff
@@ -31,6 +31,7 @@
 #include "CryptoKeyUsage.h"
 #include "ExceptionOr.h"
 #include "JsonWebKey.h"
+#include "ScriptExecutionContext.h"
```

Comment:
Curious why this is needed, since nothing in the header has changed to require more type information about `ScriptExecutionContext`. Is it to fix the unified build?

Reply:
aah moving it to where it belongs!

Reply:
I have now added a forward decl for this. And added the header in another file where I needed it.